README.md

```diff
@@ -205,54 +293,93 @@ Add an artifact name to print all of its versions instead:
 
 ```console
 $ gto show churn
-╒════════════╤═══════════╤═══════════╤═════════════════════╤═══════════════════╤══════════════╕
-│ artifact   │ version   │ stage     │ created_at          │ author            │ ref          │
-╞════════════╪═══════════╪═══════════╪═════════════════════╪═══════════════════╪══════════════╡
-│ churn      │ v3.1.0    │ staging   │ 2022-07-13 15:25:41 │ Alexander Guschin │ churn@v3.1.0 │
-│ churn      │ v3.0.0    │ prod, dev │ 2022-07-09 00:19:01 │ Alexander Guschin │ churn@v3.0.0 │
-╘════════════╧═══════════╧═══════════╧═════════════════════╧═══════════════════╧══════════════╛
+╒════════════╤═══════════╤══════════════╤═════════════════════╤══════════════╕
+│ artifact   │ version   │ stage        │ created_at          │ ref          │
+╞════════════╪═══════════╪══════════════╪═════════════════════╪══════════════╡
+│ churn      │ v3.1.0    │ staging, dev │ 2022-07-14 10:33:53 │ churn@v3.1.0 │
+│ churn      │ v3.0.0    │ prod         │ 2022-07-09 19:27:13 │ churn@v3.0.0 │
+╘════════════╧═══════════╧══════════════╧═════════════════════╧══════════════╛
+```
+
+#### Enabling multiple versions in the same Stage workflow
```

Comment:
We have decided not to support this, right?

Reply:
Yes, it's not supported in Studio and it is not how things work by default in GTO. I'm still keep this as options to get users opinion on this earlier. If someone asks about those workflows, we can redirect to this CLI options, and if they find these flags useful, we will have a valuable feedback to design other workflows (either as standalone if we decide to use a special git tag format, or as org-wide options in Studio).

Reply:
Hmm. Then let's change the text further to indicate that at the moment, we don't really see these as common workflows. Instead of `In some cases, you want to see ...`, we could say `In case / If you want to see ...`.

Reply:
Thanks, applying your suggestion!